dev/AppNotifications/NotificationProperties.cpp

```diff
@@ -57,6 +57,36 @@ NotificationProperties::NotificationProperties(winrt::AppNotification const& toa
             m_toastConferencingConfig = winrt::make_self<NotificationConferencingConfig>(config);
         }
     }
+
+
+    m_notificationType = ToastABI::NotificationType::NotificationType_Toast;
+}
+
+NotificationProperties::NotificationProperties(Microsoft::Windows::BaseNotifications::BaseNotification const& baseNotification)
+{
+    // Extract payload and convert it from XML to a byte array
+    auto payloadAsSimpleString = Helpers::WideStringToUtf8String(baseNotification.Payload());
+
+    m_payload = wil::unique_cotaskmem_array_ptr<byte>(static_cast<byte*>(CoTaskMemAlloc(payloadAsSimpleString.size())), payloadAsSimpleString.size());
```

Comment:
code changes done

Reply:
Hi Jon, I have kept the original format because, if the conversion happens in the constructor, we are sure that it will happen only once, but if we delay the conversion, it will happen twice one in get_Payload and one in get_PayloadSize which increases the cost. So, keeping the code as it is to handle the scenario